compiler/build.gradle

```diff
@@ -100,7 +100,9 @@ model {
                 } else if (osdetector.os == "windows") {
                     linker.args "-static", "-lprotoc", "-lprotobuf", "-static-libgcc", "-static-libstdc++",
                             "-s"
-                } else {
+		} else if (osdetector.arch == "ppcle_64") {
+		    linker.args "-Wl,-Bstatic", "-lprotoc", "-lprotobuf", "-Wl,-Bdynamic", "-lpthread", "-s"
+		} else {
```

Comment:
The indentation is still incorrect. Ideally it should show only 2 lines added before line 103 but because of your changes it shows deleting line 103 and adding 3 lines (103-105). Because of mismatched indentation the `else` clause also aligns with the wrong `if` clause. Take a look at the diff shown by github and you will see.

Reply:
Hi @sanjaypujare does it look okay now with the current indentation ?

Reply:
Yes. Merged already

Reply:
thanks @sanjaypujare